packages/module/src/controller/QuickStartTaskHeader.scss

```diff
@@ -1,16 +1,4 @@
 .pfext-quick-start-task-header {
-  display: flex;
```

Comment:
there is a small margin difference between the header list items. I think if you add these deleted lines 2-8 back in, it'll be good

Reply:
I'll merge this PR and make the change in my PR, thanks for the contribution! 👍

Reply:
Thanks @jschuler